forms-flow-web/src/components/Draft/List.js

```diff
@@ -79,6 +80,7 @@ export const DraftList = React.memo(() => {
   }, [dispatch, currentPage, countPerPageRef]);
 
   const onYes = () => {
+    setDraftDeleted(true);
```

Comment:
you can toggle back once the request is complete, so the button will be active if a network error happens

Reply:
actually when the delete happens if its an error or not the Modal will be closing now. need to change if the request success then only need to close that so toggle back the value in ```finally ```

Reply:
either we can avoid this toggle back because here we are asking for confirmation right so the modal will close after confirmation. then if its an error it will throw the message exactly it seems good

Reply:
👍

Reply:
Similar change was done for form delete before right? Should not send Multiple delete requests.